apps/design/frontend/src/election_info_screen.tsx

```diff
@@ -106,9 +110,53 @@ function ElectionInfoForm({
           disabled={!isEditing}
         />
       </FormField>
+      <FormField label="Seal">
+        {electionInfo.seal && (
+          <img
+            src={`data:image/svg+xml;base64,${Buffer.from(
+              electionInfo.seal
+            ).toString('base64')}`}
+            alt="Seal"
+            style={{ maxWidth: '10rem', marginBottom: '1rem' }}
+          />
+        )}
+        {(isEditing || !electionInfo.seal) && (
+          <FileInputButton
+            accept="image/svg+xml"
+            onChange={(e) => {
+              const file = e.target.files?.[0];
+              if (!file) {
+                return;
+              }
+              const reader = new FileReader();
+              reader.onload = (e2) => {
+                const svgContents = e2.target?.result;
+                if (typeof svgContents === 'string') {
+                  const seal = DomPurify.sanitize(svgContents, {
+                    USE_PROFILES: { svg: true },
+                  });
+                  setElectionInfo({ ...electionInfo, seal });
+                }
+              };
+              reader.readAsText(file);
+            }}
+            disabled={!isEditing}
+          >
+            Upload Seal Image
+          </FileInputButton>
+        )}
+      </FormField>
```

Comment:
Would be curious to see what the form + form interaction looks like. Not blocking and I'm happy to wait for a more holistic tour of the latest UI later on

Reply:
Oh duh! Got so into the habit of refactors that I forgot to add a video lol

Reply:
Just added to PR description

Reply:
Haha makes sense - the UX looks good!